src/main/java/org/apache/sysds/common/Builtins.java

```diff
@@ -86,6 +86,7 @@
 	DIAG("diag", false),
 	DISCOVER_FD("discoverFD", true),
 	DROP_INVALID("dropInvalid", false),
+	DROP_INVALID_LENGTH("dropInvalidLen", false),
```

Comment:
Please, make it consistent: `dropInvalidType`, and `dropInvalidLength` with consistent semantics, both drop the entries. Initially, I thought it would be better to return a 0/1 matrix indicating violations (or even better just return the string length per cell), but right now we could not eliminate them via other operations and thus should directly drop them (before feeding the frames into a tranformencode)